install/kubernetes/cilium/README.md

```diff
@@ -24,8 +24,6 @@ Helm chart for Cilium
 | bandwidthManager | bool | `false` |  |
```

Comment:
Nit: `during upgrade during upgrade` in the commit message for `helm: Remove default options to support upgrade`.